chart-service/chart-generator.js

```
@@ -100,6 +195,66 @@
       ctx.fillRect(0, 0, width, height);
     }
 
+    // Ensure font family is specified in the chart options
+    if (chart.options && chart.options.plugins) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)